GitCommands/Git/GitGpgController.cs

```diff
@@ -232,13 +232,13 @@ public string GetCommitVerificationMessage(GitRevision revision)
                 return null;
             }
 
-            var module = GetModule();
+            // '--raw' returns info in stderr
```

Comment:
'git verify-tag --raw' returns the info in stderr, but the command itself is redundant here.
--raw returning in stderr is just an unexpected behavior that I want to document, commands should normally only use stderr for displaying errors.

I moved the comment to the return that accesses stderr instead.

Reply:
https://github.com/gitextensions/gitextensions/pull/10261#discussion_r996349904

https://github.com/gitextensions/gitextensions/pull/10261#discussion_r996346945

My point is it doesn't matter if it is --raw or not.

Reply:
https://github.com/git/git/blob/d420dda0576340909c3faff364cfbd1485f70376/gpg-interface.c#L656-L666

See how they output to both streams?  It is not an error if error stream has output. That is what the error code provides.  I have seen a few programs do this where they output a your summary or details 8n standard output and verbose logging in error or a different result set in error. As seen above they have the potential to output a different verify output if the flag is set in standard output. Tte error is the output we expect. I certainly see the issue of they should have done it the other way around.

Reply:
The comment is intended to describe why stderr is parsed instead of stdout.
In general commands should only use stderr to display error messages to the user. I tried to document why stderr was explicitly used with this commit.

Can you make a suggestion on how to describe this behavior?

Reply:
https://github.com/git/git/blob/d420dda0576340909c3faff364cfbd1485f70376/gpg-interface.c#L661
```c++
        if (flags & GPG_VERIFY_VERBOSE && sigc->payload)
```

Typically that is the case but it is not always the case. Looks like it outputs the raw payload to stdoutput if ```GPG_VERIFY_VERBOSE``` is set.